package.json

```diff
@@ -23,7 +23,7 @@
     "rimraf": "~2.1.4"
   },
   "dependencies": {
-    "decompress-zip": "0.3.2",
+    "decompress-zip": "^0.3.2",
```

Comment:
This change is made so that upcoming updates >= 0.3.2 <0.4.0 will automatically be installed